docs/blog/content/material/ss_5.0.0beta.en.md

```diff
@@ -1,117 +1,117 @@
-+++
-title = "Following 6 months of development Apache ShardingSphere 5.0.0-beta has been officially released!"
-weight = 14
-chapter = true
-+++
-
++++
+title = "Following 6 months of development Apache ShardingSphere 5.0.0-beta has been officially released!"

```

Comment:
These are not the content that I changed, it may be caused by the different line breaks. It was not `CRLF` before?